exporter/collector/logs.go

```diff
@@ -631,3 +652,17 @@ func fixUTF8(s string) string {
 	}
 	return buf.String()
 }
+
+func bytesFromValue(v pcommon.Value) ([]byte, error) {
```

Comment:
Why not put the `json.Unmarshal` in the function too? Then the signature can just be `unmarshalAttribute(v pcommon.Value, out any) error`

Reply:
Done in [992e990](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/794/commits/992e99041c6f46520f344b0da0cb006b98b972ad)